css/style.css

```diff
@@ -57,6 +61,48 @@ a:hover {
   padding: 15px 20px;
 }
 
+.mobile-menu-background {
+  background: #e5e5e5;
+  height: 100vh;
+  z-index: 1;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+}
+
+.menu-mobile-div {
+  position: relative;
+  background: #fff;
+  height: 100vh;
+  border-bottom-left-radius: 95px;
+  border-bottom-right-radius: 95px;
+}
```

Comment:
- [ ] While implementing the changes requested by the previous reviewer, You changed the close(X) button styles. Please use the correct font-weight and size for the close(X) and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167481285-a0f70402-c031-42e1-9bb1-d53a59723c64.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167481191-4e61c776-4a4f-42c6-879e-f91a07f6c8dd.png)

Reply:
@thecodechaser Thanks for your reviews, but I have a few queries, In figma design, it has used a vector image not font-awesome, which has been requested by the previous reviewer. Now, if I am trying to change the font weight of the icon and it's not happening. I have gone through google and nothing was found to reduce font-awesome icon font-weight, so please guide me on the same.

Reply:
@RitobrotoMukherjee You can use anything you want but make sure the button matches the design. You can also use an image if you want,

Reply:
@thecodechaser Thanks for your reply, but this is confusing right! the old reviewer says to use font awesome. with which the design cannot be matched. Now you are suggesting using the design in the image! The last design before the old reviewer was exactly the same as the Figma font weight. He wanted me to replace the design of the ```X``` character and use font-awesome.
Trust me I am totally confused here! Please guide me on the same

Reply:
@RitobrotoMukherjee The previous reviewer requested the right changes you should not use the heading for the close(X) button. And you can use anything meaning font, image, or button tag. It is not good practice to use heading for buttons.

Reply:
@RitobrotoMukherjee The previous reviewer suggest to use font-awsome but you can use anything you want if that follow the best code practices.

Reply:
@thecodechaser Ok now understood! Thanks for the guidance